web/pdf_viewer.js

```diff
@@ -2186,16 +2191,32 @@ class PDFViewer {
     this.currentScaleValue = newScale;
   }
 
-  updateContainerHeightCss() {
-    const height = this.container.clientHeight;
-
+  #updateContainerHeightCss(height) {
```

Comment:
Nit: Maybe ensure that the `height` will always default to a meaningful value, since the old method didn't accept any parameters?
```suggestion
  #updateContainerHeightCss(height = this.container.clientHeight) {
```